### PR TITLE
[NCL-1810] Use PNC ide-config style

### PR DESCRIPTION
**Pull request by thescouser89**
Style from: https://raw.githubusercontent.com/project-ncl/ide-config/master/eclipse/jboss-java-formatter.xml

Note: no new code was added in this commit. It's just application of the
codestyle

### CODE REVIEW

File: src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java [hunk 1]
```diff
@@ -88,7 +77,8 @@ public void submit(Runnable task) {
         backOffRunnable.setRunnable(internalTask);
         backOffRunnable.setCancelHook(() -> unsubscribe(subscription, UnsubscribeReason.NO_DATA_FROM_SOURCE));
 
-        ScheduledFuture<?> scheduledFuture = executor.scheduleAtFixedRate(backOffRunnable, 0, backOffRunnableConfig.getPollIntervalMillis(), TimeUnit.MILLISECONDS);
+        ScheduledFuture<?> scheduledFuture = executor.scheduleAtFixedRate(backOffRunnable, 0,
+                backOffRunnableConfig.getPollIntervalMillis(), TimeUnit.MILLISECONDS);
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java]: When parameters are in multiple lines, there have to be one parameter per line.

File: src/main/java/org/jboss/pnc/bifrost/endpoint/Rest.java [hunk 2]
```diff
@@ -24,36 +24,26 @@
...
-            @QueryParam("prefixFilters") String prefixFilters,
-            @QueryParam("afterLine") Line afterLine,
-            @QueryParam("direction") Direction direction,
-            @QueryParam("maxLines") Integer maxLines,
-            @QueryParam("follow") boolean follow,
-            @QueryParam("timeoutProbeString") String timeoutProbeString); //if string is defined the server is sending given string as a connection probe. The string is printed with the logs.
+    Response getAllLines(@QueryParam("matchFilters") String matchFilters, @QueryParam("prefixFilters") String prefixFilters,
+            @QueryParam("afterLine") Line afterLine, @QueryParam("direction") Direction direction,
+            @QueryParam("maxLines") Integer maxLines, @QueryParam("follow") boolean follow,
+            @QueryParam("timeoutProbeString") String timeoutProbeString); // if string is defined the server is sending given
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/bifrost/endpoint/Rest.java]: It's been more readable before the change.

File: src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java (hunk 1 above)
Comment (thescouser89) [reply src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java]: Proposed fix upstream: https://github.com/project-ncl/ide-config/pull/1

File: src/main/java/org/jboss/pnc/bifrost/endpoint/Rest.java (hunk 2 above)
Comment (thescouser89) [reply src/main/java/org/jboss/pnc/bifrost/endpoint/Rest.java]: Proposed fix upstream: https://github.com/project-ncl/ide-config/pull/1

**Comment by thescouser89** [comment]
@matejonnet Thanks for the input! PR updated with proposed changes in the project-ncl/ide-config repository. Let me know what you think!

**Pull request closed without merging**